### PR TITLE
fix: Update image version in documentations

### PR DESCRIPTION
**Pull request by yec-akamai**
## 📝 Description

Bump all outdated versions of images to the latest in the example snippets.

### CODE REVIEW

File: docs/data-sources/vlans.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Provides details about Linode VLANs.
 ```terraform
 resource "linode_instance" "my_instance" {
   label      = "my_instance"
-  image      = "linode/ubuntu18.04"
+  image      = "linode/ubuntu23.10"
```
Comment (zliang-akamai) [inline docs/data-sources/vlans.md:20]: Can we change it to `22.04` for a longer supported time?
You can check Ubuntu lifecycle here: https://ubuntu.com/about/release-cycle

File: docs/data-sources/instances.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Each Linode instance will be stored in the `instances` attribute and will export
...
+* `image` - An Image ID to deploy the Disk from. Official Linode Images start with linode/, while your Images start with `private/`. See [images](https://api.linode.com/v4/images) for more information on the Images available for you to use. Examples are `linode/debian12`, `linode/fedora39`, `linode/ubuntu22.04lts`, `linode/arch`, and `private/12345`. See all images [here](https://api.linode.com/v4/linode/images) (Requires a personal access token; docs [here](https://developers.linode.com/api/v4/images)). *This value can not be imported.* *Changing `image` forces the creation of a new Linode Instance.*
```
[the remark this answers is not in the document]
Comment (yec-akamai) [reply docs/data-sources/instances.md]: Good catch!

File: docs/data-sources/vlans.md (hunk 1 above)
Comment (yec-akamai) [reply docs/data-sources/vlans.md]: Sure, thanks for pointing it out!

**Review by zliang-akamai — APPROVED** [review]
LGTM!

**Comment by lgarber-akamai** [comment]
@yec-akamai Would you mind also bumping the image versions in the `examples` directory?

**Review by lgarber-akamai — APPROVED** [review]
LGTM, nice work!

**Pull request merged**